http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellAppInstall.scala

```diff
@@ -0,0 +1,149 @@
+                               azureApplicationInsightsService: AzureApplicationInsightsService[F]
+)(implicit
+  F: Async[F]
+) extends AppInstall[F] {
+
+  override def databases: List[Database] =
+    List(
+      Database("cromwell", allowAccessForAllWorkspaceUsers = false),
+      Database("cbas", allowAccessForAllWorkspaceUsers = false),
+      Database("tes", allowAccessForAllWorkspaceUsers = false)
+    )
+
+  override def buildHelmOverrideValues(
+    params: BuildHelmOverrideValuesParams
+  )(implicit ev: Ask[F, AppContext]): F[Values] = for {
+    ctx <- ev.ask
+
+    // Resolve batch account in Azure
+    batchAccount <- azureBatchService.getBatchAccount(params.landingZoneResources.batchAccountName, params.cloudContext)
+
+    // Resolve application insights in Azure
+    applicationInsightsComponent <- azureApplicationInsightsService.getApplicationInsights(
+      params.landingZoneResources.applicationInsightsName,
+      params.cloudContext
+    )
+
+    // Storage container is required for Cromwell app
+    storageContainer <- F.fromOption(
+      params.storageContainer,
+      AppCreationException("Storage container required for Cromwell app", Some(ctx.traceId))
+    )
+
+    // Databases required for Cromwell App
+    dbNames <- F.fromOption(toCromwellAppDatabaseNames(params.databaseNames),
+                            AppCreationException("Database names required for Cromwell app", Some(ctx.traceId))
+    )
+
+    // Postgres server required for Cromwell App
+    postgresServer <- F.fromOption(params.landingZoneResources.postgresServer,
+                                   AppCreationException("Postgres server required for Cromwell app", Some(ctx.traceId))
+    )
+
+    // Get the pet userToken
+    tokenOpt <- samDao.getCachedArbitraryPetAccessToken(params.app.auditInfo.creator)
+    userToken <- F.fromOption(
+      tokenOpt,
+      AppCreationException(s"Pet not found for user ${params.app.auditInfo.creator}", Some(ctx.traceId))
+    )
+
+    values = List(
+      // azure resources configs
+      raw"config.resourceGroup=${params.cloudContext.managedResourceGroupName.value}",
+      raw"config.batchAccountKey=${batchAccount.getKeys().primary}",
+      raw"config.batchAccountName=${params.landingZoneResources.batchAccountName.value}",
+      raw"config.batchNodesSubnetId=${params.landingZoneResources.batchNodesSubnetName.value}",
+      raw"config.drsUrl=${drsConfig.url}",
+      raw"config.landingZoneId=${params.landingZoneResources.landingZoneId}",
+      raw"config.subscriptionId=${params.cloudContext.subscriptionId.value}",
+      raw"config.region=${params.landingZoneResources.region}",
+      raw"config.applicationInsightsConnectionString=${applicationInsightsComponent.connectionString()}",
+
+      // relay configs
+      raw"relay.path=${params.relayPath.renderString}",
+
+      // persistence configs
+      raw"persistence.storageResourceGroup=${params.cloudContext.managedResourceGroupName.value}",
```

Comment:
A fair number of these are no longer needed - probably makes more sense to leave that cleanup to a future PR, though, and leave this one as a no-op in terms of app install definition.

Reply:
Yeah, agreed -- I compared helm values before/after this PR and tried to make everything consistent (modulo the workload identity changes).